docs/azure_arc.md

```diff
@@ -0,0 +1,48 @@
+# Deploying to Azure Arc
+
+This template supports deployment to Azure Arc. The following document lists the requirements to setup Azure Arc to support the template, as well as how you deploy the application to an Arc-enable Kubernetes Cluster.
+
+- [Deploying to Azure Arc](#deploying-to-azure-arc)
+  - [Prerequisites](#prerequisites)
+  - [Regions and resource group support](#regions-and-resource-group-support)
+    - [Azure Arc enabled App Service](#azure-arc-enabled-app-service)
+
+## Prerequisites
+
+In order to deploy this template to Azure Arc, you need to have the Arc-enabled Kubernetes Cluster created. This template assumes this has already been done. To learn more about configuring an Arc environment for App Service, please see this [blog](https://aka.ms/ArcEnabledAppServices-Build2021-Blog).
+
+It's also requred to have Arc-enabled dataservices Data Controller installed in the Arc Cluster, please see more [here](https://docs.microsoft.com/en-us/azure/azure-arc/data/create-data-controller). This repository assumes a directly connected data controller.
+
+## Regions and resource group support
+
+[Please check here for latest information](https://docs.microsoft.com/en-us/azure/azure-arc/kubernetes/quickstart-connect-cluster) 
+
+### Azure Arc enabled App Service and Data Services
+
+Following these guidelines will deploy the webapi to an ARC-enabled Kubernetes cluster.
+
+To deploy from local environment run the deployment script
+
+```
+./deploy/scripts/install.sh \
+   --resource-name-prefix <resource group name> \
+   --environment-tag <name tag for all resources> \
+   --resource-group-tag <tag> \
+   --location <location (eastus or westeurope)> \
+   --node-env development \
+   --kube-environment-id <kubeEnvironmentId> \
+   --custom-location-id <customLocationId> \
+   --data-controller-id <dataControlleId>
```

Comment:
Do we think deploy.sh is a better name than install?

Reply:
It might be, but it's already in-use in the existing template. I believe renaming woudl be it's owne work item, if we want to do that.